aa_modules/aamod_epifromnifti.m

```diff
@@ -145,13 +145,27 @@
             if iscell(V), V = cell2mat(V); end
             spm_file_merge(char({V(numdummies+1:end).fname}),finalepis,0,DICOMHEADERS{1}.volumeTR);
         end
+        
+        
         % And describe outputs
+
+        % new: output streams are renameable so that other modalities
+        % (e.g., dot) can reuse aamod_epifromnifti w/ just a header alias
+        
+        outputstreams = aas_getstreams(aap,'output'); 
+        
+        ostream_01 = outputstreams{1};
+        ostream_02 = outputstreams{2};
+        ostream_03 = outputstreams{3};
```

Comment:
Here, I suggest adding some heuristics rather than hardcoding the order of all streams to make it more robust. E.g., the "dummyscans" can be assumed to be fixed across modalities, so you can search for it in the stream list.
Then we can prescribe that the main stream (epi or dot) MUST be the first stream, while the header stream MUST be called <main stream>_header. In this way, only the first stream MUST be at a fixed place.